external/solr/cores/status/conf/schema.xml

```diff
@@ -25,5 +25,7 @@ under the License.
 	<field name="status" type="string" indexed="true" stored="true" required="true"/>
 	<field name="nextFetchDate" type="pdate" stored="true" indexed="true"/>
 	<dynamicField name="metadata*" type="string" indexed="true" stored="true" multiValued="true"/>
+	<field name="key" type="string" stored="true" indexed="true"/>
```

Comment:
@mvolikas why not set the key field to multiValued = false?
I assume the dynamic field is to allow users to set a different name than 'key'

Reply:
Yes, that's the idea behind the dynamic field, and you are right, "key" should have `multiValued = false` too. thanks for pointing it out!